doc/release-notes-31384.md

```diff
@@ -0,0 +1,11 @@
+- Node and Mining
+
+---
+
+- PR #31384 fixed an issue where the coinbase transaction weight was being reserved in two different places.
+ The fix ensures the reservation happens in one place, but does not change the previous behaviour by setting the default
+ to the previous value of `8,000`.
+ However, this PR offers the advantage of overriding the default with a user-desired value and allow creating blocks
```

Comment:
I'm a bit confused by this second sentence. How does the behaviour introduced here allow for overriding the default?